app/components/quick-filter.js

```diff
@@ -20,38 +20,45 @@ export default Component.extend({
       case 'today':
         newStartDate = moment().toISOString();
         newEndDate = moment().toISOString();
+        this.set('disableClear', false);
```

Comment:
Instead of disabling clear 5 times in 5/6 cases just do it once at the start and set it as true in the case where it's not needed.